src/BenchmarkDotNet.Exporters.Plotting/BenchmarkDotNet.Exporters.Plotting.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>

```

Comment:
Why do we need net6.0 and net8.0? I think netstandard2.0 should cover everything, I don't see any conditional compilation in your code.

Reply:
@timcassell now just netstandard2.0